src/scaffolding/master.scaffolding.hbs

```diff
@@ -13,4 +13,21 @@
             <button class="splash__button splash__button--blue">See Docs</button>
         </div>
     </div>
-</header>
\ No newline at end of file
+</header>
+
+<footer class="footer">
+    <div class="swooshFoldWrapper">
```

Comment:
We need to talk about this implementation I think. I'm not sure if I approve of this but we can merge it for now.

Reply:
Let's talk 👍